client/components/checklist/style.scss

```diff
@@ -138,19 +132,21 @@
 		bottom: 0;
 		left: 34px;
 		border-left: 1px solid var( --color-neutral-100 );
+		z-index: 1;
```

Comment:
Should we avoid hard-coded z-indices in favor of using the logic from [this file](https://github.com/Automattic/wp-calypso/blob/3b3dc15cf9818f469b9efa72c410248d7d7084eb/assets/stylesheets/shared/functions/_z-index.scss)?